.travis.yml

```diff
@@ -1,33 +1,66 @@
 language: cpp
 sudo: false
-compiler: clang
-#    - clang
-#    - gcc
+compiler:
+    - clang
+    - gcc
 
-os: osx
-#    - linux
-#    - osx
+os:
+    - linux
+    - osx
+#osx_image: xcode7
+dist: trusty
+
+addons:
+  apt: 
+   sources:
+      - boost-latest
+      - ubuntu-toolchain-r-test
+   packages:
+      - gcc-4.8
+      - g++-4.8
+      - libboost1.55-all-dev
+      - libilmbase-dev
+      - libopenexr-dev
+      - libgif-dev
+      - libtiff4-dev
+      - libopenjpeg-dev
+      - libwebp-dev
+      - ffmpeg
```

Comment:
What determines if you list something in sources, packages, or in the install_homebrew_deps.bash?